src/cli/gentest.py

```diff
@@ -284,10 +269,7 @@ def __init__(self, node_config: Config.RemoteNode):
         Initialize the RequestManager with specific client config.
         """
         self.node_url = node_config.node_url
-        headers = {
-            "CF-Access-Client-Id": node_config.client_id,
-            "CF-Access-Client-Secret": node_config.secret,
-        }
+        headers = node_config.rpc_headers
```

Comment:
This will raise an error if the user doesn't add and `rpc_headers` entry to their config (as it's optional). We could check if this entry is present and set it to an empty dictionary `{}` if not.